### ISSUE TITLE
Report Request timing metrics for the SDK

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#3584]
Based on the discussion in this doc: [instrument sdk timing measurements](https://docs.google.com/document/d/1t3XewcpwnH9asnuT-f2akTd6d_j74iIJlK98GnxVHE4/edit#bookmark=id.h7fqfwn8mh0b) we have determined that we want to measure:

* Request Timing requests. In places such as fetching the experiment data, etc

This depends on the completion of SDK-69

Note: this desciption has been updated from the original content based on the doc referenced above.  A follow-up ticket will be filed for measuring startup times in the client.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/SDK-1)
┆Issue Number: SDK-1

**Comment by data-sync-user** [comment]
➤ Travis Long commented:

This will be possible once this bug gets resolved: https://bugzilla.mozilla.org/show_bug.cgi?id=1672982

**Comment by data-sync-user** [comment]
➤ Janet Dragojevic commented:

This should be broken out into individual issues before starting.
